### DISCUSSION TITLE
Tenders Search API - Query parameter value

### DISCUSSION THREAD

**Discussion by udaykeith** [OP-TED/eForms-SDK#551] [Q&A] [answered]
Hi, I am trying to search for tenders which fulfill certain criteria via /api/v3.0/notices/search. However, I am not sure how to structure the value for the "query" argument. It seems that no matter what I pass as the value for "query" I receive the same response. 

This is currently the code I am using:

```
import requests
import json

url = 'https://ted.europa.eu/api/v3.0/notices/search'  
api_key = TED_API_KEY 
query_ = 'notice-type=[cn-standard or pin-cfc-social]'

headers = {
    'Authorization': f'Bearer {TED_API_KEY}',
    'fields': "['ND', 'PD', 'CONTENT']",
    "query": "blah blah"
}


response = requests.get(url, headers=headers)
```

I receive a valid JSON with a 1000 results with this query. I have a query setup in Expert search and I used the same search query in my code but got the same results.

**Comment by bertrand-lorentz** [comment, marked as answer]
This is a question related to the search API of the TED website, so I suggest to contact their support helpdesk: https://ted.europa.eu/TED/misc/contact.do

A few hints, based on my limited knowledge of this API:
The various parameters "query", "fields", etc. have to be passed either in the URL query string for a GET request, or in the request body for a POST request. It seems you are sending them in the headers, which is not supported.

**Reply by udaykeith** [reply]
Hi,
I was able to pass the fields in the URL (had a look at the Swagger page) and I was able to get the correct response. 

Thanks!
